.github/workflows/build-and-test.yml

```diff
@@ -22,20 +25,32 @@ jobs:
         agent: [1, 2, 3]
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/setup
+      - name: Checkout all commits
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Derive appropriate SHAs for base and head for `nx affected` commands
+        uses: nrwl/nx-set-shas@v3
+      - name: Set up dependencies
+        uses: ./.github/actions/setup
 
       - name: Start Nx Agent ${{ matrix.agent }}
-        run: yarn nx-cloud start-agent
+        run: npx nx-cloud start-agent
 
   build:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/setup
+      - name: Checkout all commits
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Derive appropriate SHAs for base and head for `nx affected` commands
+        uses: nrwl/nx-set-shas@v3
+      - name: Set up dependencies
+        uses: ./.github/actions/setup
 
-      - run: yarn nx-cloud start-ci-run
+      - run: npx nx-cloud start-ci-run
 
       - name: Run Affected lint
         run: yarn nx affected:lint --parallel --maxParallel=3
```

Comment:
can we let it run with `--quiet`? I think we can also increase the `maxParallel` arg for the linting & testing steps.

![image](https://user-images.githubusercontent.com/4904455/196992626-1fad6908-2ffe-46bf-b8d9-9f5b383186bf.png)

Reply:
> I think we can also increase the maxParallel arg for the linting & testing steps.

Hosted GitHub runners only have 2 vCPUs so 2 parallel tasks would be a better fit as this workflow isn't using Nx's reusable workflows for DTE.

Reply:
> can we let it run with --quiet?

It would be better to have 0 warnings than ignore them. I usually add `--max-warnings=0` when linting to have lint warnings break the build.

The reason I switched from `yarn nx-cloud` to `npx nx-cloud` is that it was hiding errors caused by a mismatched version of `@nrwl/nx-cloud`.

Reply:
Ofc it would be better. But finding errors in that mess is currently the bigger problem I'd say. As for me, I'd still go with quiet for now